interface/models.py

```diff
@@ -137,6 +137,9 @@ def update_state(self):
             self.state = response.json()['state']
             self.save()
 
+        if self.state == self.STATE_DONE and self.review_message:
+            self.state = "reviewed"
```

Comment:
Currently `self.state` is a cache of the [vmck job state](https://github.com/vmck/vmck/blob/445dceae47748db454659f8d450dea63afd31f3c/vmck/models.py#L15-L19), let's leave it like that.

How about we display the review state separately? Where do you want to see it?

Reply:
I think the best place would be at the ```Status``` field.
I am thinking to add a flag at the ```Submission``` class level and send it to the template - and depending on that flag write ```sub.status``` or ```reviewed```.

What do you think?

Reply:
How about we display "done" if it's finished running, and then "done, reviewed" after it's reviewed?